### ISSUE TITLE
Running workflow 

### ISSUE THREAD

**Issue by muhammadrisma** [comfyanonymous/ComfyUI#5996]
### Expected Behavior

Model in folder 'checkpoints' with filename 'epicRealism.safetensors' not found. (Fix this)

### Actual Behavior

Model in folder 'checkpoints' with filename 'epicRealism.safetensors' not found.

### Steps to Reproduce

[workflow-comfyui---easily-smooth-face-skin-QGVQwgBvlvoLHlpuMZzR-cgtips-openart.ai (1).json](https://github.com/user-attachments/files/18078203/workflow-comfyui---easily-smooth-face-skin-QGVQwgBvlvoLHlpuMZzR-cgtips-openart.ai.1.json)

### Debug Logs

```powershell
Model in folder 'checkpoints' with filename 'epicRealism.safetensors' not found.
```

**Comment by LukeG89** [comment]
Do you have the model in your `ComfyUI\models\checkpoints` folder?

**Comment by LukeG89** [comment]
@vitalikr44 You pressed "R" to refresh? The recently added checkpoint model should appear in the checkpoint loader

**Comment by vitalikr44** [comment]
Yes, I pressed "R", as you asked. It didn't help.
The name of my file should be like this: "epicrealism_naturalSinRC1VAE.safetensors"

This is a quote from the report:
- **Exception Message:** Model in folder 'checkpoints' with filename 'epicrealism_naturalSinRC1VAE.safetensors' not found.
## Stack Trace

I double-checked the correctness of the file name.

**Comment by LukeG89** [comment]
> I'm new to this program. Can this model be replaced with another one? and in which node should I change the link to another model? I suspect that the contents of the file are corrupted or outdated.

Yes, you can replace the checkpoint model with another one, just put the model you want in `ComfyUI\models\checkpoints` folder, press "R" in ComfyUI and then select the model in the Checkpoint loader
